src/Components/CreateImageWizard/steps/Registration/components/ManualActivationKey.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+
+import { Button, Content, FormGroup, Popover } from '@patternfly/react-core';
+import { ExternalLinkAltIcon, HelpIcon } from '@patternfly/react-icons';
+
+import { ACTIVATIONKEYS_INFO_URL } from '../../../../../constants';
+import { useAppDispatch, useAppSelector } from '../../../../../store/hooks';
+import {
+  changeActivationKey,
+  changeOrgId,
+  selectActivationKey,
+  selectOrgId,
+} from '../../../../../store/wizardSlice';
+import { ValidatedInput } from '../../../ValidatedInput';
+
+const ActivationKeyPopover = ({ label }: { label: string }) => {
+  return (
+    <>
+      {label}
+      <Popover
+        headerContent='About activation key'
+        position='right'
+        minWidth='30rem'
+        bodyContent={
+          <Content>
+            <Content>
+              Activation keys assist you in registering and configuring systems.
+              Metadata such as role, system purpose, and usage can be
+              automatically attached to systems via an activation key, and
+              monitored with Subscription Watch.
+            </Content>
+            <Button
+              component='a'
+              target='_blank'
+              variant='link'
+              icon={<ExternalLinkAltIcon />}
+              iconPosition='right'
+              isInline
+              href={ACTIVATIONKEYS_INFO_URL}
+            >
+              Learn more about activation keys
+            </Button>
+          </Content>
+        }
+      >
+        <Button
+          icon={<HelpIcon />}
+          variant='plain'
+          className='pf-v6-u-pl-sm pf-v6-u-pt-0 pf-v6-u-pb-0'
+          aria-label='About Organization ID'
+          isInline
+        />
+      </Popover>
+    </>
+  );
+};
+
+const OrgIdPopover = ({ label }: { label: string }) => {
+  return (
+    <>
+      {label}
+      <Popover
+        headerContent='About Organization ID'
```

Comment:
Should we include info about how/where to find the org id?

Reply:
Ya I initially had this but there were a couple of places it can be done so it got a bit messy.

Will try see how we can try manage this

Reply:
I merged the two popovers together since they're so closely related. I put a popover with the same text next to both fields and the copy was taken from the red hat docs and console dot.

Also I updated the popover field slightly so it's more consistent with the [patternfly docs](https://www.patternfly.org/components/forms/form#basic)